### PR TITLE
Feature/cosmos primary db

### PR DESCRIPTION
**Pull request by jj-perry**

### CODE REVIEW

File: cosmos-db/mongo-init.js [hunk 1]
```diff
@@ -0,0 +1,2021 @@
...
+      opening_day_tuesday: true,
+      opening_day_wednesday: false,
+      opening_day_thursday: false,
+      opening_day_friday: false,
+      opening_day_saturday: false,
+      opening_day_sunday: false,
+      opening_hours_tuesday: "10:00 - 12:00",
+    },
+  },
+  metadata: {
```
Comment (paul-burnett) [inline cosmos-db/mongo-init.js]: These are now stored as `declaration` instead of `metadata`?

File: cosmos-db/mongo-init.js [hunk 2]
```diff
@@ -0,0 +1,2021 @@
...
+db.suppliers.insertOne({
+  _id: 1,
+  supplier_name: "Test Supplier",
+  supplier_url: "test-supplier",
+  local_council_urls: ["cardiff", "bath"],
+});
+
+db.createCollection("configVersion");
+db.configVersion.insertOne({
+  _id: "1.7.0",
```
Comment (paul-burnett) [inline cosmos-db/mongo-init.js]: Worth updating this to latest prod configVersion number?

Comment (paul-burnett) [reply cosmos-db/mongo-init.js]: Would also need to update the FE .env to match

File: cosmos-db/mongo-init.js (hunk 1 above)
Comment (jj-perry) [reply cosmos-db/mongo-init.js]: Updated

File: cosmos-db/mongo-init.js (hunk 2 above)
Comment (jj-perry) [reply cosmos-db/mongo-init.js]: Done

**Review by paul-burnett — APPROVED** [review]

**Pull request merged**